sabnzbd/skintext.py

```diff
@@ -862,6 +862,7 @@
     "Glitter-logText": TT(
         "All usernames, passwords and API-keys are automatically removed from the log and the included copy of your settings."
     ),
+    "Glitter-sortCompleted": TT("Sort by Completed <small>Most&rarr;Least</small>"),
```

Comment:
Don't you think `Sort by percentage remaining` or something like that is better?

Reply:
Sort by Comp% ?

Reply:
Sort by % downloaded?